### PR TITLE
Radix UI Switch

### PR DESCRIPTION
**Pull request by sean-brydon**
Making all switches consistent - using Raxix UI design instead of headless

## What does this PR do?

Replaces switch from headless UI with the reusable component that has already been made

Fixes # (issue)
https://github.com/calcom/cal.com/issues/2073 - Note Menu component is being made by @leog 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by leog** [comment]
I was wondering if we should try to move UI components to the separate UI monorepo package while we refactor/improve them. That way we can end up with all of them moved out progressively without forcing it. I was thinking about this as I'm tackling Dropdown and Dialog using Button which was already moved. Thoughts @zomars?

**Comment by zomars** [comment]
> I was wondering if we should try to move UI components to the separate UI monorepo package while we refactor/improve them. That way we can end up with all of them moved out progressively without forcing it. I was thinking about this as I'm tackling Dropdown and Dialog using Button which was already moved. Thoughts @zomars?

Agreed, let's make a rule that if someone touches a component inside `apps/web/components/ui/` he should be responsible to migrate it to `packages/ui/`.

**Review by leog — COMMENTED** [review]
Looks good, the Switch just need to be in `packages/ui` to go for the monorepo approach complying with the rule mentioned by @zomars in his comment 😉

File: packages/config/tailwind-preset.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ module.exports = {
     "./pages/**/*.{js,ts,jsx,tsx}",
     "./components/**/*.{js,ts,jsx,tsx}",
     "./ee/components/**/*.{js,ts,jsx,tsx}",
+    "../../packages/ui/**/*.{js,ts,jsx,tsx}",
```
Comment (sean-brydon) [inline packages/config/tailwind-preset.js:6]: As mentioned in Leog's PR this is the easiest approach

File: apps/web/components/booking/TimeOptions.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 // TODO: replace headlessui with radix-ui
```
Comment (zomars) [inline apps/web/components/booking/TimeOptions.tsx:1]:
```suggestion
```

**Review by zomars — APPROVED** [review]

**Review by alannnc — APPROVED** [review]
It's working great on my end

**Pull request merged**